README.md

```diff
@@ -1,2 +1,77 @@
 # obs-service-bazel_repositories
-OBS Source Service to download dependencies of projects using Bazel.
+OBS Source Service to download dependencies of projects using Bazel. This allows
+offline builds of them on OBS.
+
+It downloads all dependencies which `bazel fetch` would and then it stores them
+in the `vendor.tar.gz` archive, which then can be used for offline builds. That
+archive contains the complete Bazel cache inside `BAZEL_CACHE` directory, which
+means that for a simple project, that archive can be used like:
+
+```
+tar -zxf vendor.tar.gz
+bazel build \
+    --repository_cache=BAZEL_CACHE \
+    //...
+```
+
+After that, Bazel should build the project without downloading any dependencies.
+
+## Usage for packagers
+
+Let's assume that you are packaging a project called `foobar` and:
+
+- your package is going to be built from `foobar.tar.gz` archive
+- it can be built with `bazel build //...` command (which means that the target
+  is `//...`)
+
+First of all, before running this service, you need to have a basic `.spec`
+file already. For our `foobar` project, it can be as simple as:
+
+```
+Name:           foobar
+Version:        0.1
+Release:        0
+Summary:        Simple project using Bazel
+License:        Apache-2.0
+URL:            https://somegithost.org/foobar/foobar
+Source0:        https://somegithost.org/foobar/foobar/archive/v0.1.tar.gz#/foobar-0.1.tar.gz
+BuildRequires:  bazel
+
+%prep
+%setup -q
+
+%build
+bazel build \
+    --repository_cache=BAZEL_CACHE \
+    //...
+
+%install
+install -D -m0755 bazel-bin/foobar %{buildroot}%{_bindir}/foobar
+
+%files
+%license LICENSE
+%doc README.md
+%{_bindir}/foobar
+
+%changelog
+```
+
+For that project, create a `_service` file containing:
+
+```
+<services>
+  <service mode="disabled" name="bazel_repositories">
+    <param name="archive">foobar-*.tar.gz</param>
+  </service>
+</services>
+```
+
+Then run:
+
+```
+osc service disabledrun
+```
```

Comment:
There was some discussion about `disabledrun` being deprecated and then maybe undeprecated when `manualrun` was added over at https://github.com/openSUSE/osc/pull/826

The documentation says that `osc service run bazel_repositories` should work, but it doesn't actually appear to work that way for me.  Specifically, `osc service X` is supposed to take arguments from the _service file if there's a definition for service X in said file, but it doesn't appear to work.  So, maybe don't change the docs until that bug is resolved. :p

Reply:
I was not familiar with `manualrun`. Thanks! I will try to fix it both for `service run` and `manualrun`.

Reply:
FWIW, `run` was fixed recently in https://github.com/openSUSE/osc/commit/cb6eaf3720fdd5206c2bc06d7bf444c6fdd14ecb

Reply:
After last push, I tried `service run` at it works for me.